packages/cardano-services/src/Utxo/DbSyncUtxoProvider/types.ts

```diff
@@ -0,0 +1,10 @@
+export interface UtxoModel {
```

Comment:
I think this name could be improved as it's not really modelling the UTxO, but nothing is coming to mind right now 😅 . Let's just add a docblock to at least highlight the fact that there will be n rows per token included in the `Value`